ci/dockerfiles/rc/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:bionic
```

Comment:
Seems like this replaces the [`concourse-docker` Dockerfile](https://github.com/concourse/concourse-docker/blob/master/Dockerfile) which was used in the previous pipeline. Maybe we should update that instead? The separate repo is kind of nice for categorizing Docker-specific issues.

Also I think there'll need to be some equivalent to how the `concourse-docker` Dockerfile wires up the worker/TSA/session signing keys.

Reply:
Oh, true! I totally missed it, thanks for pointing that out, I think it indeed makes sense to keep it there! 

Going that way (not keeping it under `concourse/concourse`'s `ci` directory), do you think it'd be nice to have a way for people to build it outside our pipeline? E.g., by just `docker build -t blabla .` there in `concourse-docker`? 

As I see, that'd mean:

1. taking an `rc` URL as an arg (maybe w/ a default to the last stable release?);
2. passing a build arg with the RC when building in our pipeline (that'd mean having `--buld-arg` available in `concourse/builder`).

Wdyt?

Reply:
Hmmm I don't think it's super important to be able to `docker build` from the `concourse-docker` repo, personally. For local development they could always build from the `concourse` repo.